rdkit-postgresql95/meta.yaml

```diff
@@ -4,23 +4,27 @@ package:
 
 source:
   git_url: https://github.com/rdkit/rdkit.git
-  git_tag: Release_2016_09_2
+  git_rev: Release_2017_03
   patches:
-    - Makefile.patch
+    - cmakelists.txt.patch
     - adapter.cpp.patch
 
 build:
   number: 1
 
 requirements:
   build:
-    - python
-    - rdkit >=2016.09
-    - postgresql95
+    - cmake
+    - msys2-conda-epoch >=20160418 [win]
+    - m2-msys2-runtime [win]
+    - m2-libiconv [win]
+    - m2-libintl [win]
+    - m2-diffutils [win]
+    - m2-patch [win]
+    - postgresql >=9.5
```

Comment:
I may not have been patient enough yesterday, but when I tried to use the standard package to build I ended up with compile problems related to use of an int128 type. Instead of investigating I just switched back to using postgresql95 to build.